ONSdigital/ras-rm-survey
# added /info and /health endpoints

# Motivation and Context
This PR is to add the /info and /health endpoints for this service.

# What has changed
* Modified `main.go` to add handling for two endpoints.
* Added the `endpoint_responses.go` file to keep endpoint-related code.
* Added the `config.go` file to store environment variables.
* Added the `endpoint_test.go` file for endpoint-related unit tests.

# How to test?
* Run the service and navigate to the two endpoints. Check that they are returning JSON responses.
* Run the tests in endpoint_test.go and make sure they pass.

# Links
[Trello card](https://trello.com/c/eTjt5q6w)
